### ISSUE TITLE
LND: Implement LSPS0

### ISSUE THREAD

**Issue by JssDWt** [breez/lspd#105]
https://github.com/BitcoinAndLightningLayerSpecs/lsp/blob/main/LSPS0/README.md

**Comment by kingonly** [comment]
This is lower priority, right? Getting lsps2 to work on CLN should be our main goal.

**Comment by JssDWt** [comment]
> This is lower priority, right? Getting lsps2 to work on CLN should be our main goal.

Yes exactly. We'll implement CLN first. Then SDK. Then at some point LND.

**Comment by JssDWt** [comment]
Closing due to inactivity
